src/validation.cairo

```diff
@@ -43,8 +43,15 @@ fn validate_target(self: @ChainState, block: @Block) -> Result<(), ByteArray> {
 }
 
 fn validate_timestamp(self: @ChainState, block: @Block) -> Result<(), ByteArray> {
-    // TODO: implement
-    Result::Ok(())
+    let timestamp: u32 = *block.header.time;
```

Comment:
Small nit: are these variables necessary?

Reply:
just updated the PR, removed all intermediary variables